app/browser/reducers/torReducer.js

```diff
@@ -15,15 +15,18 @@ const torReducer = (state, action) => {
     case appConstants.APP_SET_TOR_NEW_IDENTITY:
       filtering.setTorNewIdentity(action.url, action.tabId)
       break
-    case appConstants.APP_ON_TOR_INIT_ERROR:
-      state = state.setIn(['tor', 'initializationError'], action.message)
-      break
-    case appConstants.APP_ON_TOR_INIT_SUCCESS:
-      state = state.setIn(['tor', 'initializationError'], false).setIn(['tor', 'percentInitialized'], null)
+    case appConstants.APP_ON_TOR_ERROR:
+      state = state.setIn(['tor', 'error'], action.message)
       break
     case appConstants.APP_ON_TOR_INIT_PERCENTAGE:
       state = state.setIn(['tor', 'percentInitialized'], action.percentage)
       break
+    case appConstants.APP_ON_TOR_ONLINE:
+      state = state.setIn(['tor', 'online'], true)
```

Comment:
this may need to clear intializationError if one exists

Reply:
i guess it's done in https://github.com/brave/browser-laptop/pull/14599/files#diff-8a9bac13ef6264feb2b6da7c18d86b3cR836

Reply:
I arranged the caller of this to also do onTorError(null).  Is that good enough?

Reply:
@riastradh-brave yup should be fine as long as this isn't called in multiple places (since i doubt we ever want the case where the tor error is shown when tor is online)

Reply:
5bdfbf0ec